Description: Autotools modifies these files.
 Patch needed to enable build twice in a row.

diff --git a/doc/stamp-vti b/doc/stamp-vti
index 68fe3a6..79dabd0 100644
--- a/doc/stamp-vti
+++ b/doc/stamp-vti
@@ -1,4 +1,4 @@
-@set UPDATED 8 February 2014
+@set UPDATED 21 February 2014
 @set UPDATED-MONTH February 2014
 @set EDITION 2.17
 @set VERSION 2.17
diff --git a/doc/version.texi b/doc/version.texi
index 68fe3a6..79dabd0 100644
--- a/doc/version.texi
+++ b/doc/version.texi
@@ -1,4 +1,4 @@
-@set UPDATED 8 February 2014
+@set UPDATED 21 February 2014
 @set UPDATED-MONTH February 2014
 @set EDITION 2.17
 @set VERSION 2.17
